Formula/astrometry-net.rb

```diff
@@ -57,6 +55,7 @@ def install
   test do
     xy = Language::Python.major_minor_version Formula["python@3.8"].opt_bin/"python3"
     ENV["PYTHONPATH"] = libexec/"lib/python#{xy}/site-packages"
```

Comment:
Maybe an idea to remove these two lines so that the test picks up the error.

Reply:
Thanks, I hadn't noticed that (despite being surprised that the tests were passing... ugh).  I removed them in the next commit.